colorcoder/Main.java

```diff
@@ -1,120 +1,30 @@
 package colorcoder;
 
-public class Main {
```

Comment:
Main.java - class name and method name semantically same ? but differ in casing - why this ambiguity 
why Main class should hold the responsibility of printing the manual

Reply:
I will rename Main class and move print method to other class where it make more sense